opentoken.js

```diff
@@ -120,9 +120,10 @@ OpenTokenAPI.prototype.createToken = function (pairs, cb) {
     return cb(new Error("OpenToken missing 'subject'"));
   }
 
-  pairs['not-before'] = now.toISOString();
-  pairs['not-on-or-after'] = expiry.toISOString();
-  pairs['renew-until'] = renewUntil.toISOString();
+  // Remove milliseconds
```

Comment:
Can you document the "why" rather than the "what":  eg `// format as "yyyy-MM-ddTHH:mm:ssZ" per opentoken spec`

Reply:
```suggestion
  // Format as "yyyy-MM-ddTHH:mm:ssZ" per OpenToken spec
```

👍  good idea!

Reply:
@73rhodes I implemented the change you suggested